npmpackagejsonlint.config.cjs

```diff
@@ -1,4 +1,12 @@
 module.exports = {
+  overrides: [
```

Comment:
Willen we dit of moeten we perfectionist vertellen dat overrides als laatste mogen (want dat kan)?